bindings/python/CMakeLists.txt

```diff
@@ -107,25 +95,10 @@ if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang")
 	endif()
 endif()
 
-execute_process(COMMAND
-  ${PYTHON_EXECUTABLE} -c "import distutils.sysconfig;
-print(';'.join(map(str, [
-	distutils.sysconfig.get_python_lib(plat_specific=True, prefix=''),
-	distutils.sysconfig.get_config_var('EXT_SUFFIX')
-])))"
-	OUTPUT_VARIABLE _python_sysconfig_vars OUTPUT_STRIP_TRAILING_WHITESPACE)
-
-list(GET _python_sysconfig_vars 0 PYTHON_SITE_PACKAGES)
-list(GET _python_sysconfig_vars 1 PYTHON_EXT_SUFFIX)
-
-message(STATUS "Python site packages: ${PYTHON_SITE_PACKAGES}")
-# python 2 does not provide the 'EXT_SUFFIX' sysconfig variable, so we use cmake default then
-if (NOT "${PYTHON_EXT_SUFFIX}" STREQUAL "None")
-	message(STATUS "Python extension suffix: ${PYTHON_EXT_SUFFIX}")
-	# we mimic the name, created by setuptools
-	# example: libtorrent.cpython-36m-x86_64-linux-gnu.so
-	set_target_properties(python-libtorrent PROPERTIES SUFFIX ${PYTHON_EXT_SUFFIX})
-endif()
+message(STATUS "Python site packages: ${Python3_SITEARCH}")
+
+get_target_property(PYTHON_EXT_SUFFIX python-libtorrent SUFFIX)
+message(STATUS "Python extension suffix: ${PYTHON_EXT_SUFFIX}")
```

Comment:
This was more of a test message, maybe it's unneeded now?

Reply:
In the end it can be removed, sure, but for now it is still useful for debugging purposes.